client/my-sites/importer/uploading-pane.jsx

```diff
@@ -16,8 +17,9 @@ import Button from 'components/forms/form-button';
 import DropZone from 'components/drop-zone';
 import ProgressBar from 'components/progress-bar';
 import Gridicon from 'components/gridicon';
+import { connectDispatcher } from './dispatcher-converter';
 
-export default React.createClass( {
+export const UploadingPane = React.createClass( {
```

Comment:
It's different because I didn't think about it when I added the name to it. Now I'm confused at what would be the best way to merge them: the short name or the verbose name…

The `displayName` was deigned to be unique, but do we care about that? It's long and not really necessary for when using and importing the component.

Thoughts @gwwar?